modules/administration/pages/public-cloud.adoc

```diff
@@ -312,7 +313,7 @@ However, it is possible to clone systems and register them successfully by delet
 # rm -rf /var/cache/salt
 ----
 +
-. If you are using a traditional registered system, clean up the working files with this command:
+. If you are using a traditionally managed client, clean up the working files with this command:
```

Comment:
```suggestion
. If you are using a traditional client, clean up the working files with this command:
```

"Traditionally managed" could be confusing. We refer to them as "traditional clients" elsewhere.